scripts/extensions/sams/src/apps/samsAttachmentsWidget.tsx

```diff
@@ -0,0 +1,255 @@
+/* eslint-disable react/no-multi-comp */
```

Comment:
we should consider putting this on [superdesk-client-code-style](https://github.com/superdesk/superdesk-client-code-style) cause I've seen it in other components